rmw_zenoh_cpp/src/detail/rmw_data_types.cpp

```diff
@@ -127,11 +118,13 @@ std::unique_ptr<saved_msg_data> rmw_subscription_data_t::pop_next_message()
   std::lock_guard<std::mutex> lock(message_queue_mutex_);
 
   if (message_queue_.empty()) {
-    // This tells rcl that the check for a new message was done, but no messages have come in yet.
+    // This tells rcl that the check for a new message was done, but no messages
```

Comment:
Here and in other places, if the code has not changed, please keep the original so that the overall diff is minimal.